.styleci.yml

```diff
@@ -0,0 +1,8 @@
+enabled:
+
+disabled:
+  - align_double_arrow
```

Comment:
but I aggree with [this post](http://mnapoli.fr/approaching-coding-style-rationally/) when it comes to whitespace alignment